app/assets/stylesheets/home.scss

```diff
@@ -73,8 +81,15 @@ html, body {
 }
 
 .large-3 {
-  margin-bottom: 25px;
   height: 220px;
+  margin: 8px 0 8px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `8px 0` instead of `8px 0 8px 0`